lib/matplotlib/axis.py

```diff
@@ -1036,11 +1033,22 @@ def _translate_tick_params(kw, reverse=False):
         to the generic tick1, tick2 logic of the axis. Additionally, there
         are some other name translations.
 
-        Returns a new dict of translated kwargs.
+        Parameters
+        ----------
+            kw
```

Comment:
The parameters list should not have additional indent wrt to the Parameters section header.

Reply:
fixed

Reply:
maybe we could use https://pypi.org/project/pydoclint/ as pre commit hook?